src/util/workers-health-monitor.test.ts

```diff
@@ -159,26 +159,26 @@ describe("workers-health-monitor", () => {
 			}));
 
 			const workerSendingImAliveToMasterInterval = setInterval(() => {
-				if ((cluster.workers[1]!.on as jest.Mock).mock.calls.length > 0) {
-					(cluster.workers[1]!.on as jest.Mock).mock.calls[0][1]();
-					(cluster.workers[2]!.on as jest.Mock).mock.calls[0][1]();
+				if ((cluster.workers?.[1]?.on as jest.Mock).mock.calls.length > 0) {
```

Comment:
Why is this?

Reply:
I mean why use `?` instead of the `!`, as the `!` still make it a "valid" test

Reply:
Using `!` was throwing in typescript errors, cause `cluster.workers` can be `undefined`.

Reply:
I'd suggest using `!` when possible, because for test code, we want a more strict behaviour, like we always want to test to fail if the value is `undefined`. Using `?`, while okay in these couple lines, may leave the test in a less strict behaviour in some other cases. 

I've tried the following should remove the complained from ts

```
                               if ((cluster.workers![1]!.on as jest.Mock).mock.calls.length > 0) {
                                       (cluster.workers![1]!.on as jest.Mock).mock.calls[0][1]();
                                       (cluster.workers![2]!.on as jest.Mock).mock.calls[0][1]();
                                }
```